test/indexes.test.ts

```diff
@@ -21,6 +21,26 @@ ava('[Indexes] Create index', async (t: Assertions) => {
 	await mongoClient.dropCollection();
 });
 
+ava('[Indexes] List all indexes', async (t: Assertions) => {
+	const collection = global.db.collection(`test-${Date.now()}`);
+	const mongoClient = new MongoClient(collection, Object, null);
+
+	await mongoClient.createIndex('index_1');
+	await mongoClient.createIndex('index_2');
+	await mongoClient.createIndex('index_3');
+
+	const createdIndexes = await (await mongoClient.findAllIndexes()).toArray();
+	const correctIndexes =
+		createdIndexes[1].name === 'index_1_1' &&
```

Comment:
could you sur assertions for that :  

t.is(createdIndexes[1].name, 'index_1_1', 'index 1 created and listed')
...